### ISSUE TITLE
Questions about pre-trained models

### ISSUE THREAD

**Issue by Baboom-l** [mileyan/Pseudo_Lidar_V2#27]
Hi, I used the your pre-trained model  'sdn_kitti_object.pth.', but the results of the test on train set are as follows：

[2021-12-18 15:26:29 main.py:214] INFO     TEST Epoch0:0        L 1.770 RLI 3.965 RLO 0.142 ABS 0.072 SQ 0.443 DEL 0.945 DELQ 0.985 DELC 0.992

For this result I checked the information in the pre-trained model and it has an RMSE of only 3.0320 which is quite different from the actual test result 3.965 , do you know what causes this situation?

**Comment by rsj007** [comment]
Why not evaluate in the eval dataset?

**Comment by Baboom-l** [comment]
@rsj007  I tested it on the validation set at first, however I found the results a bit strange, the pre-trained model felt like it was not fine-tuned, so I tested it on the training set again before asking the question
